rel/app/standalone.exs

```diff
@@ -0,0 +1,176 @@
+defmodule Standalone do
+  @moduledoc false
+  require Logger
+
+  @doc """
+  Copies ERTS into the release.
+  """
+  @spec copy_erlang(Mix.Release.t()) :: Mix.Release.t()
+  def copy_erlang(release) do
+    {erts_source, erts_bin_dir, erts_lib_dir, erts_version} = erts_data()
+
+    erts_destination_source = Path.join(release.path, "erts-#{erts_version}/bin")
```

Comment:
Perhaps we should copy to a directory without the version? This would simplify a bit the Elixir and other parts. :)